tests/UI/README.md

```diff
@@ -31,6 +31,8 @@ npm install
 
 ### Playwright parameters
 
+| Parameter           | Description                                          |
```

Comment:
Without this, https://github.com/PrestaShop/PrestaShop/tree/develop/tests/UI#upgrade-test is not rendered correctly